apps/test/integration/levelSolutions/applab/ec_canvas.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
         'rect(80, 120, 160, 240);\n' +
         'setStrokeWidth(3);\n' +
         'setStrokeColor("red");\n' +
-        'setFillColor("yellow");\n' +
+        'setFillColor(rgb(255,0,0));\n' +
```

Comment:
This will only validate that it doesn't throw exceptions.

I suspect that our test coverage of the API is not great ATM. One thing you could potentially write unit tests for is  `applabCommands.rgb`, i.e. maybe just two simple cases - one for with an alpha, one for without. Also, what happens if you call rgb with only 1-2 args?